pkg/cmd/experimental/router/router.go

```diff
@@ -144,6 +145,7 @@ func RunCmdRouter(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg *
 			return cmdutil.UsageError(cmd, "You may not pass negative labels in %q", cfg.Labels)
 		}
 		label = valid
+		nodeSelector = label
```

Comment:
Hmm, ok. I will add a new optional flag called ```--selector```. That way it also matches up with the one in the ```ipfailover``` command.